Dashboard/va.gov.artemis.commands/Dsio/Pregnancy/DsioSavePregDetailsToOtherNamespaceCommand.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System.Collections.Generic;
+using VA.Gov.Artemis.Vista.Broker;
+using VA.Gov.Artemis.Vista.Utility;
+
+namespace VA.Gov.Artemis.Commands.Dsio.Pregnancy
+{
+    /// <summary>
+    /// A command to save pregnancy details to a different namespace in VistA
+    /// </summary>
+    public class DsioSavePregDetailsToOtherNamespaceCommand : DsioCommand
+    {
+        /// <summary>
+        /// Creates the command
+        /// </summary>
+        /// <param name="newBroker">An object which allows communication with VistA and implements IRpcBroker</param>
+        public DsioSavePregDetailsToOtherNamespaceCommand(IRpcBroker newBroker) : base(newBroker) { }
+
+        /// <summary>
+        /// The Ien of the person, either created or updated
+        /// </summary>
+        public string Ien { get; set; }
+
+        /// <summary>
+        /// The name of the RPC
+        /// </summary>
+        public override string RpcName
+        {
+            get { return "WVRPCOR SAVEDATA"; }
+        }
+
+        /// <summary>
+        /// Add command arguments to the RPC call. 
+        /// Call prior to calling "Execute"
+        /// </summary>
+        /// <param name="pregnancy"></param>
+        public void AddCommandArguments(DsioPregnancy pregnancy, string patientDfn, bool isPatientPregnant)
```

Comment:
Ok, I like this one much better.  Why have the "differentnamespace" function above even or did that just get leftover from something else?